src/renderers/WebGLRenderer.js

```diff
@@ -192,11 +191,15 @@ function WebGLRenderer( parameters ) {
 		_canvas.addEventListener( 'webglcontextlost', onContextLost, false );
 		_canvas.addEventListener( 'webglcontextrestored', onContextRestore, false );
 
-		_gl = _context || _canvas.getContext( _webglversion, contextAttributes ) || _canvas.getContext( 'experimental-webgl', contextAttributes );
+		var webglVersion = _webgl2 && typeof WebGL2RenderingContext !== 'undefined' ? 'webgl2' : 'webgl';
+
+		_gl = _context || _canvas.getContext( webglVersion, contextAttributes );
```

Comment:
Do you think we should try to create WebGLContext if we fail to create WebGL2Context?